fms_fsdp/utils/config_utils.py

```diff
@@ -55,6 +55,26 @@ def get_model_config(model_variant):
             nheads=16,
             nlayers=24,
         )
+    elif model_variant == "8b":
+        llama_config = LLaMAConfig(
+            src_vocab_size=128256,
+            emb_dim=4096,
+            nheads=32,
+            kvheads=8,
+            nlayers=32,
+            hidden_grow_factor=3.5,
+            max_expected_seq_len=8192,
+        )
```

Comment:
Do we not have support for these model configs already in main FMS? Should we be adding these here?

Reply:
`get_model` is slightly complex to this use case.   the remote FMS repo was also not the latest.  so for configuration I prefer doing it separately here.

Reply:
Oh, fair enough. We should probably reconcile these at some point though